rosapi/src/rosapi/params.py

```diff
@@ -37,11 +37,13 @@
 from rcl_interfaces.msg import Parameter, ParameterType, ParameterValue
 from rcl_interfaces.srv import ListParameters
 import rclpy
+import rospy
```

Comment:
This import is incorrect and has broken the rosapi node... clearly we need more testing in CI 😓